graphql_resolver.go

```diff
@@ -206,10 +214,12 @@ func (s *SessionResolver) PlayerSessions() ([]*PlayerSessionResolver, error) {
 
 // PlayerSessionResolver struct
 type PlayerSessionResolver struct {
+	player    PlayerResolver
 	playerID  graphql.ID
```

Comment:
if you're gonna get rid of playerId from the schema then you don't need this. You'll also need to delete the resolver method for that field. I don't recommend you delete the playerID field though.